<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndoorPositioningSystemUI</class>
 <widget class="QMainWindow" name="IndoorPositioningSystemUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1708</width>
    <height>996</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>2136</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IndoorPositioningSystem</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_Play_Pause">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>880</y>
      <width>41</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_Duration">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>840</y>
      <width>1141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_Current_Time">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>840</y>
      <width>67</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Total_Time">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>840</y>
      <width>67</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Video">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>1281</width>
      <height>721</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Video_Frame_ID">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Video_Frame_Timestamp">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>32</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame Timestamp:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_ID_1">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>80</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tag ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_timestamp_1">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>79</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Timestamp:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor1_ID_1">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>110</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Anchor ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor2_Distance_1">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>130</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Video_Frame_ID_value">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Video_Frame_Timestamp_value">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>31</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_ID_value_1">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>81</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_timestamp_value_1">
    <property name="geometry">
     <rect>
      <x>1536</x>
      <y>80</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor1_Distance_value_1">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>110</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor2_Distance_value_1">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>130</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_ID_value_2">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>213</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor2_Distance_2">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>260</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_ID_2">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>210</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tag ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor1_Distance_value_2">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>240</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor2_Distance_value_2">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>260</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor1_Distance_2">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>240</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_timestamp_2">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>210</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Timestamp:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_timestamp_value_2">
    <property name="geometry">
     <rect>
      <x>1537</x>
      <y>212</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_ID_value_3">
    <property name="geometry">
     <rect>
      <x>1393</x>
      <y>342</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_ID_3">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>340</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tag ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor1_ID_3">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>380</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Anchor ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_timestamp_3">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>339</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Timestamp:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_timestamp_value_3">
    <property name="geometry">
     <rect>
      <x>1538</x>
      <y>342</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_UWB_Data_Analysis">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>490</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Analyse UWB Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Video_Stream">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>31</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Information about the video frame</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 13pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>Video Stream:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>20</y>
      <width>21</width>
      <height>851</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_Video_Stream_2">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>30</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Information about the UWB data</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 13pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>UWB Data Stream:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_UWB_Localization">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>530</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Visualize localization</string>
    </property>
    <property name="text">
     <string>UWB Localization Schema</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Export_Data">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>900</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Export data for Pixel to Real model. This includes UWB coordinates synchronized with video frames</string>
    </property>
    <property name="text">
     <string>Export Data Frame by Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Predict_Model">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>920</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Start pixel to real model for coordinates prediction</string>
    </property>
    <property name="text">
     <string>Start Pixel-to-real</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Predict_Optical">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>879</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Start optical method for coordinates estimation</string>
    </property>
    <property name="text">
     <string>Start Optical method</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_measurement_time_1">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>160</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Measurement time (ms):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_measurement_time_value_1">
    <property name="geometry">
     <rect>
      <x>1512</x>
      <y>163</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_measurement_time_2">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>290</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Measurement time (ms):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_measurement_time_value_2">
    <property name="geometry">
     <rect>
      <x>1514</x>
      <y>292</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_measurement_time_3">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>440</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Measurement time (ms):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_measurement_time_value_3">
    <property name="geometry">
     <rect>
      <x>1514</x>
      <y>442</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Predict_coordinates">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>880</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Predict coordinates by Optical method:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_UWB_Show_Coordinates">
    <property name="geometry">
     <rect>
      <x>1530</x>
      <y>530</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Visualize localization</string>
    </property>
    <property name="text">
     <string>Show UWB Coordinates</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Predict_coordinates_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>920</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Predict coordinates by Pixel-to-Real method:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Optical_Show_Coordinates">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>879</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Visualize localization</string>
    </property>
    <property name="text">
     <string>Show Coordinates</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Pixel_to_Real_Show_Coordinates">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>920</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Visualize localization</string>
    </property>
    <property name="text">
     <string>Show Coordinates</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor1_Distance_1">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>110</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor1_ID_value_1">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>110</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor2_ID_1">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>132</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Anchor ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor2_ID_value_1">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>131</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>190</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>30</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor2_ID_2">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>262</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Anchor ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor2_ID_value_2">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>261</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor1_ID_2">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>240</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Anchor ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor1_ID_value_2">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>239</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>320</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor2_ID_3">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>400</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Anchor ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor2_ID_value_3">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>400</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor1_ID_value_3">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>379</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor1_Distance_value_3">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>381</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor2_Distance_3">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>400</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor1_Distance_3">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>380</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Anchor2_Distance_value_3">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>400</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>470</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1708</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Video"/>
    <addaction name="actionLoad_human_detector_weights"/>
    <addaction name="actionLoad_model_params"/>
    <addaction name="actionLoad_intrinsic_calibration_params"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Video">
   <property name="text">
    <string>Open video and UWB data</string>
   </property>
   <property name="toolTip">
    <string>The folder should include video data as well as UWB data</string>
   </property>
  </action>
  <action name="actionOpen_UWB_data">
   <property name="text">
    <string>Open UWB data</string>
   </property>
  </action>
  <action name="actionLoad_model_params">
   <property name="text">
    <string>Load pixel to real model params</string>
   </property>
  </action>
  <action name="actionLoad_intrinsic_calibration_params">
   <property name="text">
    <string>Load camera intrinsic params</string>
   </property>
  </action>
  <action name="actionLoad_human_detector_weights">
   <property name="text">
    <string>Load human detection model weights</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
